src/tests/functional/plugin/shared/include/snippets/check_broadcast.hpp

```diff
@@ -25,7 +25,7 @@ typedef std::tuple <
 > CheckBroadcastParams;
 
 class CheckBroadcast : public testing::WithParamInterface<CheckBroadcastParams>,
-                virtual public ov::test::SnippetsTestsCommon {
+                virtual public SnippetsTestsCommon {
```

Comment:
Minor: if you update this code, let's fix tabs please 😃

Reply:
As you wish 😄 Done, thanks